extensions/interactions/interaction_specs.json

```diff
@@ -85,6 +85,13 @@
         "content_id": null,
         "unicode_str": "Type an expression here, using only numbers."
       }
+    }, {
+      "description": "Use fraction instead of the division symbol (÷) to represent division.",
```

Comment:
Suggested alternative wordings:

- Either add "a" before "fraction", or
- Change to "Represent division using fractions (rather than ÷)."

Reply:
Done, thanks!